shared/components/Clickable/Clickable.jsx

```diff
@@ -23,11 +24,13 @@ Clickable.propTypes = {
   dangerouslyAddClassName: PropTypes.string,
   dangerouslyAddStyle: PropTypes.object,
   children: PropTypes.node.isRequired,
+  type: PropTypes.string,
```

Comment:
There are actually known, valid values for `type` (https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button).

PropTypes must use `oneOf` instead of an arbitrary string when there are a distinct set of possible values.

However, I recommend to just remove it as a prop since nothing is actually using that interface. Let's not add the prop until we actually need to. Just hardcode `type="button"` into the JSX and call it a day!